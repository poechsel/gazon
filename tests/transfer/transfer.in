login u1
pass p1
ls 
get server_file.txt
ls
put client_file.txt 74
ls
