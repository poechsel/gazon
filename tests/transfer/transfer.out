total 4
-rw-r--r-- 1 <*> <*>  136 <*> server_file.txt
total 4
-rw-r--r-- 1 <*> <*>  136 <*> server_file.txt
total 8
-rw-r--r-- 1 <*> <*>  74 <*> client_file.txt
-rw-r--r-- 1 <*> <*> 136 <*> server_file.txt
